modules/shortcodes/instagram.php

```diff
@@ -139,40 +138,35 @@ function jetpack_instagram_handler( $matches, $atts, $url ) {
 	}
 
 	if ( ! empty( $response_body->html ) ) {
-		if ( ! $did_script ) {
-			$did_script = true;
-			add_action( 'wp_footer', 'jetpack_instagram_add_script' );
-		}
-
-		// there's a script in the response, which we strip on purpose since it's added above
+		wp_enqueue_script( 'jetpack-instagram-embed', plugins_url( 'js/instagram.js', __FILE__ ), array( 'jquery' ) );
```

Comment:
Yeah we updated this on the WP.com side, and I've now updated this PR as well. Thanks for reviewing this.